cmd/mt-whisper-importer-writer/main.go

```diff
@@ -182,11 +181,13 @@ func main() {
 
 	cluster.Init("mt-whisper-importer-writer", gitHash, time.Now(), "http", int(80))
 
+	cassandra.ConfigProcess()
```

Comment:
again. config validation (ConfigProcess()) should be *before* we start initalizing stuff. the sooner we can abort the program the better.